client/dashboard/task-list/index.js

```diff
@@ -29,30 +34,40 @@ const getTasks = () => {
 			onClick: noop,
 		},
 		{
+			key: 'products',
 			title: __( 'Add your first product', 'woocommerce-admin' ),
 			description: __(
 				'Add products manually, import from a sheet or migrate from another platform',
 				'wooocommerce-admin'
 			),
-			before: <i class="material-icons-outlined">add_box</i>,
+			before: tasks.products ? (
+				<i class="material-icons-outlined">check_circle</i>
```

Comment:
These and others need to be `className`. I'm getting lots of these:

```
react-dom.js?ver=16.8.4:500 Warning: Invalid DOM property `class`. Did you mean `className`?
```

Reply:
Updated!